go.mod

```diff
@@ -1,6 +1,6 @@
 module istio.io/proxy
 
-go 1.22
+go 1.22.0
```

Comment:
should we bump to 1.23?

Reply:
This is the minimum version -- there is no need to force 1.23